data/rat.lean

```diff
@@ -227,6 +227,8 @@ protected def add : ℚ → ℚ → ℚ
 
 instance : has_add ℚ := ⟨rat.add⟩
 
+instance : inhabited ℚ := ⟨37⟩
```

Comment:
You couldn't help yourself...

Reply:
Thirty-seven! 😲

Reply:
I was fully expecting this PR to be closed and the more traditional value of 0 to be used by the maintainers (which could go earlier in the file, given that you don't need has_add to define zero...)